gossip/group.go

```diff
@@ -242,3 +240,29 @@ func (g *group) addInfo(i *info) (contentsChanged bool, err error) {
 	err = util.Errorf("info %+v not added to group", i)
 	return
 }
+
+func (g *group) toProto() *proto.GossipGroup {
```

Comment:
we generally want to avoid this kind of thing (unless i'm misunderstanding why it's necessary). gogoproto gives us enough flexibility in defining our protos that we want to use our proto objects as the in-memory representation as well.